packages/desktop-client/src/components/reports/reports/CashFlowCard.tsx

```diff
@@ -170,57 +393,24 @@ export function CashFlowCard({
             />
             <DateRange start={start} end={end} />
           </View>
-          {data && (
-            <View style={{ textAlign: 'right' }}>
-              <PrivacyFilter activationFilters={[!isCardHovered]}>
-                <Change amount={income - expenses} />
-              </PrivacyFilter>
-            </View>
-          )}
+          {dataOk &&
+            (meta?.isSimpleView || meta?.isSimpleView === undefined
+              ? retViewCondensed(isCardHovered, income, expenses)
+              : retViewDetailed(
+                  totalIncome,
+                  totalExpenses,
+                  totalTransfers,
+                  isCardHovered,
+                ))}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏instead of calling functions here - these should be components.